# Array of keypoint 3D locations.

Header header
Keypoint3D[] keypoints_3D
